tools/cbmc/proofs/TaskPool/TaskDelete/README.md

```diff
@@ -0,0 +1,19 @@
+This proof demonstrates the memory safety of the TaskDelete function.  The
+initialization for the task to be delete and `pxCurrentTCB` is quite similar
+(since the task handle may be NULL, and in that case `pxCurrentTCB` is used).
+The task to be deleted requires the stack in the task control block to be
+allocated, and the flag for static allocation to be properly set (i.e., valid
+values as defined by the macros) Task lists are initialized with these tasks
+and nondet. filled with a few more items.  We assume a nondet. value for
```

Comment:
Missing full stop after parenthesis and then confusing use of "." in abbreviation. Spell out "non-deterministic" instead.